cmd/client/wallet/main.go

```diff
@@ -92,6 +92,7 @@ var (
 	transferReceiverPtr   = transferCommand.String("to", "", "Specify the receiver account")
 	transferAmountPtr     = transferCommand.Float64("amount", 0, "Specify the amount to transfer")
 	transferShardIDPtr    = transferCommand.Int("shardID", 0, "Specify the shard ID for the transfer")
+	transferToShardIDPtr  = transferCommand.Int("toShardID", 0, "Specify the shard ID for the transfer")
```

Comment:
Specify the destination shard ID for the cross shard transfer

Reply:
thanks for checking!